src/client/render2d/svg/group.tsx

```diff
@@ -7,8 +7,22 @@ import { Group as GroupGeometry } from '../object/group'
 import { GeometryView, toSvgTransform } from './rendering'
 
 type Props = { model: GroupGeometry, world: Transform }
-export const Group = observer(({ model: { children, transform }, world }: Props) => (
-  <g transform={toSvgTransform(transform)}>
-    {children.map((obj, i) => (<GeometryView key={i} obj={obj} world={world.clone().then(transform.inverse())}/>))}
-  </g>
-))
+export const Group = observer(({ model: { children, transform }, world }: Props) => {
+
+  // If we have the identity transform forgo the group to save on dom elements
+  const elems = children.map((obj, i) => (<GeometryView key={i} obj={obj} world={transform.clone().then(world)}/>))
+
+  if (transform.isIdentity()) {
+    return (
+      <>
+        {elems}
+      </>
+    )
+  } else {
+    return (
+      <g transform={toSvgTransform(transform)}>
+        {elems}
+      </g>
+    )
+  }
+})
```

Comment:
some ⛳️ ing, also pulled out the `objWorld` calculation since it is constant for the map call:

```ts
type Props = { model: GroupGeometry, world: Transform }
export const Group = observer(({ model: { children, transform }, world }: Props) => {
  const objWorld = transform.clone().then(world)
  const elems = children.map((obj, i) => (<GeometryView key={i} obj={obj} world={objWorld}/>))
  return transform.isIdentity()
    ? <>{elems}</> // If we have the identity transform forgo the group to save on dom elements
    : <g transform={toSvgTransform(transform)}>{elems}</g>
})
```